release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -58,6 +58,13 @@ The `coreos-installer` utility now has `iso customize` and `pxe customize` subco
 [id="ocp-4-10-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-10-AWS-default-components"]
+==== New default component types for AWS installations
+
+The {product-title} 4.10 installer uses new default component types for installations on AWS. The installer uses the following components by default:
+* AWS EC2 m6i instances for both control plane and compute nodes, where available
+* AWS EBS gp3 storage
```

Comment:
Suggested:
```suggestion
The {product-title} 4.10 installer uses new default component types for installations on AWS. The installer uses the following components by default:

* AWS EC2 M6i instances for both control plane and compute nodes, when available on the region, otherwise the fallback M5 instances will be used.
* AWS EBS gp3 storage for both control plane and compute nodes
```

Not sure if we could link to the [official page](https://aws.amazon.com/ec2/instance-types/m6i/) with a brief like this:
> Amazon EC2 M6i instances are powered by 3rd generation Intel Xeon Scalable processors and deliver up to 15% better price performance compared to M5 instances. M6i instances feature a 4:1 ratio of memory to vCPU, just like M5 instances and support up to 128 vCPUs per instance, which is 33% more than M5 instances.

wdyt @staebler  @sdodson ?

Reply:
Formatting fixed -- thanks

Reply:
@ctauchen the instance name (`M6i`, `M5`), officially, has a capital letter on the first letter, when the doc refers to the instance size (`m6i.xlarge`) it will be lower case. So from `m6i` to `M6i`

Reply:
Yeah, I wouldn't bother explaining why they're better. It's a new default, they get them if they don't care, if they do care then I'd expect them to know what the differences are or where to find the details.

Reply:
> @ctauchen the instance name (`M6i`, `M5`), officially, has a capital letter on the first letter, when the doc refers to the instance size (`m6i.xlarge`) it will be lower case. So from `m6i` to `M6i`, as I pointed out in the comment above. wdyt?

Agreed and fixed. Thanks!